src/Castle.Core.Tests/project.json

```diff
@@ -19,13 +19,18 @@
   },
   "compilationOptions": {
     "warningsAsErrors": true,
-    "define": [ "FEATURE_XUNITNET" ]
+    "define": [ "FEATURE_XUNITNET" ],
+    "keyFile": "../../buildscripts/CastleKey.snk"
   },
+  "resource": [
+    "Core.Tests/Ressources/MoreRes/TestRes.resx"
```

Comment:
You've spelt `Resources` wrong so that isn't working. I can also see the resx file has references to System.Windows.Forms so I suspect that won't work either on .NET Core. I actually got rid of some of this sort of thing already and replaced it, depending on if this is actually testing code using ResourceManager we should just replace it with something cleaner.